### ISSUE TITLE
[Bug]: Doc is wrong about primary key field support data type

### ISSUE THREAD

**Issue by binbinlv** [milvus-io/milvus#20543]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Environment

```markdown
- Milvus version: latest
- Deployment mode(standalone or cluster):both
- SDK version(e.g. pymilvus v2.0.0rc2):latest
- OS(Ubuntu or CentOS): 
- CPU/Memory: 
- GPU: 
- Others:
```

### Current Behavior

Doc is wrong about primary key field support data type
![image](https://user-images.githubusercontent.com/83755740/201564303-94e84466-2fcf-41b2-a5a8-ea4e22bba30d.png)

https://milvus.io/docs/v2.1.x/schema.md#Supported-data-type

### Expected Behavior

primary key field only support two kinds of data type:
INT64: numpy.int64
VARCHAR: VARCHAR

### Steps To Reproduce

```markdown
int8_field = FieldSchema(name="int8", dtype=DataType.INT8, is_primary=True)
float_field = FieldSchema(name="float", dtype=DataType.FLOAT)
bool_field = FieldSchema(name="bool", dtype=DataType.BOOL)
string_field = FieldSchema(name="string", dtype=DataType.VARCHAR, max_length=65535)
float_vector = FieldSchema(name="float_vector", dtype=DataType.FLOAT_VECTOR, dim=dim)
>>> schema = CollectionSchema(fields=[int8_field, float_field, bool_field, float_vector])
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/binbin/embedded-test-env-3.8/lib/python3.8/site-packages/pymilvus/orm/schema.py", line 55, in __init__
    raise PrimaryKeyException(0, ExceptionsMessage.PrimaryKeyType)
pymilvus.exceptions.PrimaryKeyException: <PrimaryKeyException: (code=0, message=Primary key type must be DataType.INT64.)>
```

**Comment by yanliang567** [comment]
@binbinlv could you please attach the link of the doc

**Comment by binbinlv** [comment]
> @binbinlv could you please attach the link of the doc

Done

**Comment by AnthonyTsu1984** [comment]
Please assign to me, thanks.

**Comment by binbinlv** [comment]
Fixed in official website doc(2.2.*, 2.1.* and 2.0.*)，and closed.
